voxblox/include/voxblox/integrator/tsdf_integrator.h

```diff
@@ -56,17 +56,23 @@ class TsdfIntegratorBase {
   struct Config {
     EIGEN_MAKE_ALIGNED_OPERATOR_NEW
 
-    float default_truncation_distance = 0.1;
-    float max_weight = 10000.0;
+    float default_truncation_distance = 0.1f;
+    float max_weight = 10000.0f;
     bool voxel_carving_enabled = true;
     FloatingPoint min_ray_length_m = 0.1;
     FloatingPoint max_ray_length_m = 5.0;
+    float voxel_weight_factor = 1.0f;
     bool use_const_weight = false;
     bool allow_clear = true;
     bool use_weight_dropoff = true;
     bool use_sparsity_compensation_factor = false;
     float sparsity_compensation_factor = 1.0f;
 
+    // The coordinate axis convention used for the optical axis. This is used
```

Comment:
could you elaborate on the optical axis. You mean this is the axis that goes forward along the optical axis of the sensor?

Reply:
yes, exactly, I adjusted the description